openmls/src/storage/kat_storage_stability.rs

```diff
@@ -0,0 +1,722 @@
+use base64::Engine;
```

Comment:
Can you add some description up here on what's happening in this file and how to use it?